ext/mro/mro.xs

```diff
@@ -397,68 +457,80 @@ mro_get_isarev(...)
     if(isarev) {
         HE* iter;
         hv_iterinit(isarev);
-        while((iter = hv_iternext(isarev)))
-            av_push_simple(ret_array, newSVsv(hv_iterkeysv(iter)));
+        while((iter = hv_iternext(isarev))) {
+            SV* ksv;
+            assert(HeKLEN(iter) != HEf_SVKEY);
+            ksv = newSVhek(HeKEY_hek(iter)); /* prev hv_iterkeysv(iter) */
```

Comment:
mro is ext/ so can use C99, you can combine the declaration and initialization.

Reply:
collapsed 2 lines